Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -476,6 +476,116 @@ public enum EdKeyAgreementAlgorithm {
 }
 
 public class EdKey {
+    public static func generatePrivateKey(algo: EdSigningAlgorithm) -> VectorUInt8 {
+        switch algo {
+        case .ed25519:
+            return Curve25519.Signing.PrivateKey().rawRepresentation.copyToVectorUInt8()
+        case .ed448:
+            return Data(count: 0).copyToVectorUInt8()
+        }
+    }
+
+    public static func generatePrivateKeyKeyAgreement(algo: EdKeyAgreementAlgorithm) -> VectorUInt8
+    {
+        switch algo {
+        case .x25519:
+            return Curve25519.KeyAgreement.PrivateKey().rawRepresentation.copyToVectorUInt8()
+        case .x448:
+            return Data(count: 0).copyToVectorUInt8()
+        }
+    }
+
+    public static func privateToPublic(algo: EdSigningAlgorithm, priv: SpanConstUInt8)
+        -> CryptoOperationReturnValue
+    {
+        var rv = CryptoOperationReturnValue()
```

Comment:
Nit:  I'm not a fan of `rv` as a variable name.  Is it a return value?

If it's already used elsewhere in this file, it's okay to ignore this.